lib/spack/spack/spec.py

```diff
@@ -709,6 +804,11 @@ def from_node_dict(node):
         for name, enabled in node['variants'].items():
             spec.variants[name] = VariantSpec(name, enabled)
 
+        # XXX(deptypes): why are dependencies not meant to be read here?
```

Comment:
Not necessarily, it was just an oddity I noticed when adding the deptypes. This looked like a place to add code to.